packages/dd-trace/src/telemetry/send-data.js

```diff
@@ -1,4 +1,21 @@
 const request = require('../exporters/common/request')
+
+const debug = process.env.DD_TELEMETRY_DEBUG
```

Comment:
Is this flag the same as we have already in the config?
[DD_TELEMETRY_DEBUG_ENABLED](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/config.js#L476)